### PR TITLE
1722-sdg-17-goals-grid

### PR DESCRIPTION
**Pull request by mlbrgl**
https://user-images.githubusercontent.com/13406362/198283712-b51ec7e8-f4b9-4d2c-be4d-43744bee5891.mp4

- demo: https://hans.owid.cloud/admin/gdocs/1UOhC3dU2Zc1DEt2Yzqblfc3l9K1ZS27fcNHQa7uwnxI/preview
- ⚠️ responsive work pending grid refactor
- [demo gdoc](https://docs.google.com/document/d/1UOhC3dU2Zc1DEt2Yzqblfc3l9K1ZS27fcNHQa7uwnxI/edit)

see #1722

### CODE REVIEW

**Comment by marcelgerber** [comment]
I guess this is what you mean by "responsive work pending", but just pointing out that this is completely broken on mobile at the moment just to be sure:
image

**Comment by mlbrgl** [comment]
@marcelgerber yep indeed 🙂 Thanks for double-checking though

**Review by ikesau — APPROVED** [review]
Nice! Just a small suggestion

Also, something about the timing of the hover transitions seems a bit off. I can't really place it, but maybe you should check in with Marwa to see if there's a way they could be made to feel smoother?

File: site/css/colors.scss [hunk 1]
```diff
@@ -73,3 +73,30 @@ $culture-color: #af488f;
...
+    12: #bf8b2e,
+    13: #3f7e44,
+    14: #0a97d9,
+    15: #56c02b,
+    16: #00689d,
+    17: #19486a,
+);
+
+:root {
+    @each $i, $sdgColor in $sdgColors {
```
[the remark this answers is not in the document]
Comment (mlbrgl) [reply site/css/colors.scss:99]: @ikesau I see what you mean, thanks for the suggestion. I'll be doing some more color-related work soon, so I'll hang onto indexes for a tiny bit longer, and switch if they eventually don't add anything.

[the remark this answers is not in the document]
Comment (mlbrgl) [reply site/css/colors.scss:99]: Regarding the transitions, these are actually inherited from leaking admin.css transitions (see #1749).
On the baked version, where admin.css is not included, there are no transitions. Checking with Marwa on override with a transition that affect both text and background color (probably what feels off)

**Pull request merged**